move2kube/exercise-3-customizations/exercise-3a-overriding-built-in-transformers/README.md

```diff
@@ -1 +1,130 @@
+    ls myproject/deploy
+    ```
+
+    If you notice, the Kubernetes yamls are generated in myproject/deploy/yamls directory.
+    
+    ```console
+    cicd                  compose               knative               knative-parameterized yamls                 yamls-parameterized
+    ```
+
+5. Lets customize the generated output. We will use a [custom configured version](https://github.com/konveyor/move2kube-transformers/tree/main/custom-dockerfile-change-built-in-behavior) of the [nodejs built-in transformer](https://github.com/konveyor/move2kube/tree/main/assets/built-in/transformers/dockerfilegenerator/nodejs) and [kubernetes built-in transformer](https://github.com/konveyor/move2kube/tree/main/assets/built-in/transformers/kubernetes/kubernetes) to achieve this. We already have these in this exercise in the `customizations` folder.
+
+6. Run Move2Kube transform with the customizations, specifying the customization using the -c flag.
+
+    ```console
+    $ move2kube transform -s ../../source/enterprise-app/ -c customizations/ --qa-skip --overwrite
+    ```
+
+7. Check the modified Dockerfile for frontend service of the `enterprise-app`:
+
+    ```console
+    $ cat myproject/source/frontend/Dockerfile
+    ```
+
+    Observe the base image is changed from `registry.access.redhat.com/ubi8/nodejs-12` to `quay.io/konveyor/nodejs-12` in the generated nodejs dockerfile.
+
+    ```console
+    FROM quay.io/konveyor/nodejs-12
+    COPY . .
+    RUN npm install
+    RUN npm run build
+    EXPOSE 8080
+    CMD sh start-nodejs.sh
+    ```
+
+    The CMD instruction in the generated dockerfile is modified to use the `start-nodejs.sh` file.
+
+    ```console
+    FROM quay.io/konveyor/nodejs-12
+    COPY . .
+    RUN npm install
+    RUN npm run build
+    EXPOSE 8080
+    CMD sh start-nodejs.sh
+    ```
+
+6. List the files in the frontend service directory of the `enterprise-app`:
+
+    ```console
+    $ ls myproject/source/frontend
+    ```
+
+    A new `start-nodejs.sh` file is added to `frontend` service directory which will be used in the CMD dockerfile instruction.
+
+    ```console
+    Dockerfile        __mocks__         jest.config.js    package-lock.json src               stylePaths.js     webpack.common.js
+    LICENSE           dist              manifest.yml      package.json      start-nodejs.sh   test-setup.js     webpack.dev.js
+    README.md         dr-surge.js       nodemon.json      server.js         stories           tsconfig.json     webpack.prod.js
+    ```
+
+
+8. List the kubernetes deployment artifacts generated by Move2Kube:
+
+    ```console
+    ls myproject/
+    ```
+
+    The default location of the kubernetes yamls that are generated changed from `myproject/deploy/yamls` to `myproject/yamls-elsewhere`.
+
+    ```console
+    Readme.md                     scripts                       yamls-elsewhere
+    deploy                        source                        yamls-elsewhere-parameterized
+    ```
+
```

Comment:
Can you please also add zipped customizations folder, so that can be directly used in the UI. We are already providing the zip of the source folder here https://github.com/konveyor/hackfest/blob/main/move2kube/source/enterprise-app.zip. 
Also, can you please add a Move2Kube UI related instruction here similar to what we have in the step 1  here https://github.com/Akash-Nayak/hackfest/blob/main/move2kube/exercise-2-cf-enterprise-app/README.md#steps-for-move2kube-cliui.

Reply:
Added!